frontend/components/WatchMyPixelForm.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div h-screen w="25svw" flex justify-center bg-black>
+    <div mx-auto mt-32 max-w-full flex flex-col items-center text-left text-gray-300>
+      <div text-center text-4xl>
+        Watch my pixel form
+      </div>
+      <div mt-8 flex flex-col items-center text-lg>
+        <form flex flex-col items-center w="15svw" @submit.prevent="submitForm">
+          <FormField name="username" :validate-on-blur="!isFieldDirty">
+            <FormItem w="full">
+              <FormControl>
+                <Input v-model="formData.name" type="text" placeholder="Name" !w="full" />
+              </FormControl>
+            </FormItem>
+          </FormField>
+          <FormField name="username" :validate-on-blur="!isFieldDirty">
```

Comment:
double username field

Reply:
its calendar field with wrong name, fixed it